internal/wrappers/grpcs/vorpal-grpc.go

```diff
@@ -0,0 +1,103 @@
+package grpcs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/checkmarx/ast-cli/internal/logger"
+	vorpalManagement "github.com/checkmarx/ast-cli/internal/wrappers/grpcs/protos/vorpal/managements"
+	vorpalScan "github.com/checkmarx/ast-cli/internal/wrappers/grpcs/protos/vorpal/scans"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+type VorpalWrapper struct {
+	grpcClient      *ClientWithTimeout
+	fullHostAddress string
```

Comment:
I don't think you need the word full. the host is enough, it's not a path

Reply:
I named it full because it contains also the port. Maybe endPoint will be better name?